### DISCUSSION TITLE
Show new missions in proper order

### DISCUSSION THREAD

**Discussion by ghost** [BG-Software-LLC/SuperiorSkyblock2#714] [Denied Feature Requests] [answered]
Right now I have this problem where the new missions doesn't display in order. For example Farmer I,II, III displays in the GUI like - Farmer II, Farmer I, Farmer III. Could you add a way so we can show them in a certain order?

**Comment by OmerBenGera** [comment, marked as answer]
Open an issue on github, its a bug.
